bundle/language_en.properties

```
@@ -785,6 +785,8 @@ angal.medicals.thismonth
 angal.medicals.today                                                                                   = Today
 angal.medicals.type                                                                                    = Type
 angal.medicals.withlot                                                                                 = With lots
+angal.medicalstock.allcharges                                                                          = All charges
```

```suggestion
angal.medicalstock.allcharges.txt                                                                      = All charges
```
change references too

---

I might also suggest that the message texts are exact once at the beginning of the file as constants so as not to have to get them repeatedly.